index.js

```diff
@@ -1,12 +1,96 @@
-try {
-    const OpenBrowser = require('./cfeScraping/index.js');
+/* eslint-disable indent */
+'use strict';
+// we're gonna use strict mode in all
 
-    'use strict';
-    // we're gonna use strict  mode in all
+require('dotenv').config();
 
-    console.log('Hello Message');
-    OpenBrowser.OpenBrowser();
+const fs = require('fs');
+const Discord = require('discord.js');
+const cooldowns = new Map();
+// eslint-disable-next-line object-curly-spacing
+const { Client, Collection, Intents } = require('discord.js');
+const client = new Client({
+  intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES],
+});
+// Assign the value "!" to the constant prefix, which you will use as the
+// bot prefix.
+const prefix = process.env.PREFIX;
 
-} catch (err) {
-    console.error('An error have been occurred:', err);
-}
\ No newline at end of file
+client.commands = new Collection();
+
+const commandFiles = fs
+  .readdirSync('./commands')
+  .filter((file) => file.endsWith('.js'));
```

Comment:
You should avoid doing this; it's not readable at all. You should try to be explicit about the functionality of your program. In this case, since you're using filesystem to read from the module, it's better to have this straightforwardly. One way can be using a map

```
const help = require('./help')
const cfe = require('./cfe')

const moduleByCommand = {
  help: help,
  cfe: cfe
}
```